jest.config.js

```diff
@@ -0,0 +1,11 @@
+const customJestConfig = {
+    setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+    testEnvironment: 'jest-environment-jsdom',
+    coverageReporters: ['json-summary', 'text', 'lcov'],
+    collectCoverageFrom: ['src/**/*.{ts,tsx}'],
```

Comment:
Let's exclude `.stories.tsx` files from here so they are not included in the coverage report